### PR TITLE
Modify template and playbook for fio test

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

**Comment by hongkailiu** [comment]
@schituku Can you please take a look?

**Comment by mffiedler** [comment]
@schituku  fyi

File: storage/fio/fio-test.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    shell: oc process -p "VOLUME_CAPACITY={{ volume_capacity }}" -p "STORAGE_CLASS_NAME={{ storage_class_name }}" -p "IMAGE={{ image }}" -p "NS_KEY={{ node_selector_key }}" -p "NS_VALUE={{ node_selector_value }}" -f "{{ tmp_folder }}/files/content/fio-pod-pv.json" | oc create --namespace="{{ test_project_name }}" -f -
+    tags:
+      - setup
+
+  - block:
+      - name: wait until the pod is running
+        shell: oc get pod -n "{{ test_project_name }}" | grep -v deploy | grep Running
+        register: pod_ready_result
+        until: pod_ready_result.rc == 0
+        retries: 5
```
Comment (schituku) [inline storage/fio/fio-test.yaml]: make retries to 10

Comment (hongkailiu) [reply storage/fio/fio-test.yaml]: 17 as we discussed.

**Comment by schituku** [comment]
This PR is verified and is good to go.

**Pull request merged**